packages/vc-verification/src/resolver/evm-data-aggregator.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  CachedDIDDocument,
+  CachedRoleCredential,
+  CachedRoleDefinition,
+  RoleEIP191JWT,
+} from '../models';
+import { IDataAggregator } from './data-aggregator';
+import { IServiceEndpoint } from '@ew-did-registry/did-resolver-interface';
+import { IRoleDefinitionV2 } from '@energyweb/credential-governance';
+import { VerifiableCredential } from '@ew-did-registry/credentials-interface';
+import type { RoleCredentialSubject } from '@energyweb/credential-governance';
+
+/**
+ * A class to aggreagate data from EVMs
+ */
+export class EVMDataAggregator implements IDataAggregator {
```

Comment:
I don't think this really needs to be implementing an interface to be honest. The sign of that is that `EthersProviderIssuerResolver` is accepting a concrete `EVMDataAggregator` and not the interface.
```suggestion
export class EVMDataAggregator {
```

Reply:
This is needed because the implementations in passport-did-auth would also need the same interface to pass their own implementation of Cache.

Reply:
Ok, I agree with @JGiter that I don't see why passport-did-auth will need a different cache implementation, but it doesn't really hurt to have an interface